config/routes.rb

```diff
@@ -1,5 +1,9 @@
+require 'sidekiq/web'
+require 'sidekiq/cron/web'
+
 Rails.application.routes.draw do
   devise_for :users
+  mount Sidekiq::Web => '/sidekiq'
```

Comment:
does anybody have access to the route? 

Could you pls add basic authentication there? only for developers, cause only devs use this tool